question27/js/news.js

```diff
@@ -0,0 +1,158 @@
+'use strict';
+
+const createElementWithClassName = (element, name) => {
+  const createdElement = document.createElement(element);
+  createdElement.classList.add(name);
+  return createdElement;
+};
+
+const newsWrapper = document.getElementById('js-newsWrapper');
+const tabsGroup = document.getElementById('js-tabs');
+const contentsWrapper = createElementWithClassName("div", "contentsWrapper");
+contentsWrapper.id = "js-contentsWrapper";
+
+if(!localStorage.getItem('token')) {
+  window.location.href = "./login.html"
+}
+
+//Get json data
+async function callApi() {
+  try {
+    const res = await fetch('https://myjson.dit.upm.es/api/bins/5oo3');
+    if (!res.ok) {
+      throw new Error(`サーバーリクエストに失敗しました: ${res.status}`);
+    }
+    const json = await res.json();
+    return json.data;
+  }
+  catch(error) {
+    console.error(error);
+  }
+  finally {
+    console.log('CONPLETE!');
+  }
```

Comment:
Q.
Is this ’finally’ necessary?
I would appreciate it if you could tell me!

Reply:
Thank you for your review!! I am sorry you told me before :bow: Fixed so please confirm.

fix commit : https://github.com/csakyo/homework/pull/41/commits/98d73171be17edd87be5b240a56cba6968730b46